exercises/exercise5.js

```diff
@@ -0,0 +1,12 @@
+// function to convert a number to its binary representation
+
+const convertNumberToBinary = (num) => {
+  return (num >>> 0).toString(2);
+};
+
+// function to calculate the Fibonacci sequence up to a given number
+
+const fibonacci = (num) => {
```

Comment:
The function name could be improved. It is more understandable if it starts with a verb.